admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html

```diff
@@ -31,15 +31,15 @@
 
     <title>${resource.jcr:title}</title>
 
-    <sly data-sly-include="styles.html"></sly>
+    <sly data-sly-include="styles.html" />
 
     <sly data-sly-list="${helper.model.siteCSS}">
         <link rel="stylesheet" href="${item}" type="text/css">
     </sly>
 
-    <link rel="stylesheet" href="/etc/felibs/admin.css" type="text/css">
-    <link rel="stylesheet" href="/etc/felibs/admin/dependencies.css" type="text/css">
-    <link rel="stylesheet" href="/etc/felibs/admin/dependencies/font-awesome/css/font-awesome.min.css" type="text/css">
+    <link rel="stylesheet" href="/etc/felibs/admin.css" type="text/css" />
```

Comment:
link tag does not have / at the end according to the specs if I am not mistaken - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link

Reply:
reverted